### PR TITLE
Update/case number

### PR DESCRIPTION
**Pull request by liuboaibc**
add case number and link

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@leoendless pls help to review this PR

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ import { ReactComponent as TwitterIcon } from '../../assets/twitter-default.svg'
 import { ReactComponent as TwitterColorIcon } from '../../assets/twitter-hover.svg'
 import WechatCode from '../../assets/wechat_code.svg'
 import GroupCode from '../../assets/group_code.svg'
+import BeiAnIcon from '../../assets/备案图标.png'
```
Comment (leoendless) [inline src/components/Footer/index.js]: please use an English word as a resource name.

**Pull request merged**